testsuite/drivers/lkt_toolbox_driver.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from drivers.base_driver import BaseDriver
+
+
+class LktToolboxDriver(BaseDriver):
+    """
+    Custom driver that will automatically run the "lkt_toolbox" utility
+    on the testcase's "test.lkt" file.
+    """
+
+    @property
+    def mandatory_files(self):
+        return ['test.lkt']
+
+    @property
+    def script_and_args(self):
```

Comment:
`lkt_toolbox` is not a script, this is misleading. I would suggest to just get rid of this property and just put the list literal in `run`.